exercises/rectangles/src/example/java/RectangleCounter.java

```diff
@@ -15,13 +15,13 @@ int countRectangles(final String[] rawGrid) {
 
     private static final class Grid {
 
-        private enum Entry {
+        private enum Element {
             CORNER,
             HLINE,
             VLINE,
             SPACE;
 
-            private static Entry fromChar(final char rawGridEntry) {
+            private static Element fromChar(final char rawGridEntry) {
```

Comment:
I'm sick, so apologies for the abbreviated reply; but your `Piece` made me think of another option: `Tile`! I'm happy to use whichever of those two you prefer :)